auth-web/src/stores/business.ts

```diff
@@ -123,7 +123,8 @@ export const useBusinessStore = defineStore('business', () => {
       requestActionCd: nr.requestActionCd,
       natureOfBusiness: nr.natureBusinessInfo,
       expirationDate: nr.expirationDate,
-      applicants: nr.applicants
+      applicants: nr.applicants,
+      corpNum: nr.corpNum
```

Comment:
I think there are a few interfaces that should be updated. I think Hongjing did something similar about a month ago with a new property from Namex API.

Reply:
Let me look at her changes and make sure I didn't miss them refactoring part of this

Reply:
corpNum was missing, that's it I think